spark-derived-feature-view/docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: "2.1"
+services:
+  spark:
+    image: apache/spark:v3.3.1
+    container_name: spark
+    user: root
+    ports:
+      - "8080:8080"
+      - "7077:7077"
+    environment:
+      - SPARK_WORKER_CORES=1
+      - SPARK_WORKER_MEMORY=1G
+      - SPARK_DRIVER_MEMORY=1G
+      - SPARK_EXECUTOR_MEMORY=500M
+    volumes:
+      - "./main.py:/tmp/main.py"
+      - "./data:/tmp/feathub-data"
+      - "spark_log:/opt/spark/log"
```

Comment:
Why do we need this?

Reply:
I used to start the container with the default user, and the default user has no permission to create the log directory when starting Spark. So I created one by adding a volumn. But now I start the container as `root`, it can be removed.